R/g.part5.definedays.R

```diff
@@ -5,10 +5,21 @@ g.part5.definedays = function(nightsi, wi, indjump, nightsi_bu,
   Nts = nrow(ts)
   lastDay = FALSE
   # define local functions ----
-  qwindow2timestamp = function(qwindow) {
+  qwindow2timestamp = function(qwindow, epochSize) {
     H = floor(qwindow)
     M = floor((qwindow - H) * 60)
     S = floor((qwindow - H - M/60) * 60 * 60)
+    expected_S = seq(0, 60, by = epochSize)
+    if (any(!S %in% expected_S)) {
```

Comment:
`!S %in% expected_S` is performed twice. It would read easier if you split it up in

```
seconds_tobe_revised = which(!S %in% expected_S)
if (length(seconds_tobe_revised) > 0) {
 for (si in seconds_tobe_revised) {
 }
}
```